client/js/table-view.js

```diff
@@ -74,10 +76,50 @@ class TableView{
         this.formulaBarEl.addEventListener('keyup', this.handleFormulaBarChange.bind(this));
     }
 
+    renderSum(collie, total) {
+        const whichColumn = document.querySelector(`#sum-${collie}`);
+        whichColumn.textContent = total;
+    }
+    
+    sumOf(arr) {
+        return arr.reduce(((total, x) => total+=x),0);
```

Comment:
Put spaces around `+=` and after the commas, please.  Visual consistency is very important, to reduce "noise" that distracts from what you are trying to say.